sandbox/run.sh

```
@@ -0,0 +1,84 @@
+echo "  Kubeconfig:  $([ -n "${KUBECONFIG_MOUNT:-}" ] && echo 'mounted (read-only)' || echo 'NOT SET')"
+echo ""
+echo "  Filesystem:  Only worktree is writable. Host is isolated."
+echo "  See docs/docker-sandbox-blast-radius.md for full security analysis."
+echo "=============================="
+echo ""
+
+docker run -it --rm \
+    --name "$CONTAINER_NAME" \
+    -v "${SANDBOX_DIR}:/sandbox" \
+    -v "${ADC_PATH}:/tmp/adc.json:ro" \
+    ${KUBECONFIG_MOUNT:-} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)